aws/resource_aws_imagebuilder_image.go

```diff
@@ -134,17 +134,22 @@ func resourceAwsImageBuilderImage() *schema.Resource {
 				Type:     schema.TypeString,
 				Computed: true,
 			},
-			"tags": tagsSchema(),
+			"tags":     tagsSchema(),
+			"tags_all": tagsSchemaComputed(),
```

Comment:
missed on first review @bflad: just need to update this resource's Read method to set these attributes

Reply:
Updated in https://github.com/hashicorp/terraform-provider-aws/pull/18723/commits/7dc456f1093f7a695dbbb67fa8dcfff3f9712875